python/docs/basics.md

```diff
@@ -103,7 +103,7 @@ of 1 GiB and a maximum size of 4 GiB. The pool uses
 ```python
 >>> import rmm
 >>> pool = rmm.mr.PoolMemoryResource(
-...     upstream=rmm.mr.CudaMemoryResource(),
```

Comment:
Just FYI, you can still use `arg_name=` for positional arguments to be more explicit. The error here was because the argument name is wrong. The following would work:

```
 pool = rmm.mr.PoolMemoryResource(
   ...: ...     upstream_mr=rmm.mr.CudaMemoryResource(),  # not "upstream"
   ...: ...     initial_pool_size=2**30,
   ...: ...     maximum_pool_size=2**32)
```

Reply:
Darn, that would have been a nicer fix.